src/loki/search.cc

```diff
@@ -410,11 +407,11 @@ struct bin_handler_t {
         correlated.edges.push_back(std::move(path_edge));
       }
       // correlate its evil twin
+      const DirectedEdge* other_edge = nullptr;
       const GraphTile* other_tile;
-      auto opposing_edge_id = reader.GetOpposingEdgeId(candidate.edge_id, other_tile);
-      const DirectedEdge* other_edge;
-      if (opposing_edge_id.Is_Valid() && (other_edge = other_tile->directededge(opposing_edge_id)) &&
-          edge_filter(other_edge) != 0.0f) {
+      auto opposing_edge_id = reader.GetOpposingEdgeId(candidate.edge_id, other_edge, other_tile);
```

Comment:
should go back inside the `if` statement so that other shortcircuites happen first

Reply:
which of them ? there is only `costing->Filter` here

Reply:
ah sorry i mixed this up with another line when i looked at the diff. sorry for that!